packages/trimerge-sync/src/AbstractLocalStore.ts

```diff
@@ -445,14 +445,14 @@ export abstract class AbstractLocalStore<CommitMetadata, Delta, Presence>
       await this.sendEvent({ type: 'ready' }, { self: true });
     });
   }
-  update(
+  async update(
     commits: Commit<CommitMetadata, Delta>[],
     presence: ClientPresenceRef<Presence> | undefined,
-  ): void {
-    if (this.closed) {
+  ): Promise<void> {
+    if (this.closed || (commits.length === 0 && !presence)) {
```

Comment:
What's the reasoning behind this change?

Reply:
We had a test validate that there are no calls if commits is empty but since this is async and it was immediately checking if the status of calls, the test was just invalid previously.